packages/discovery-provider/src/models/model_utils.py

```diff
@@ -74,7 +74,7 @@ def validate_field_helper(field, value, model, field_type):
     # TODO: need to write custom validator for these datetime fields as jsonschema
     # validates datetime in format 2018-11-13T20:20:39+00:00, not a format we use
     # also not totally necessary as these fields are created server side
-    if field in ("created_at", "updated_at"):
+    if field in ("created_at", "updated_at", "release_date"):
```

Comment:
this change was crucial and might fix having to use `parse_release_date` elsewhere. without this, the release_date would always appear as `None` and be reset to `created_at` due to: `Validation: Setting the default value None for field release_date of type ['string', 'null'] because of error: Instance {'release_date': datetime.datetime(2023, 12, 19, 4, 38, 14)} is not proper. Errors: [\"datetime.datetime(2023, 12, 19, 4, 38, 14) is not of type 'string', 'null'\"]`